app/models/note.rb

```diff
@@ -4,5 +4,10 @@ class Note < ApplicationRecord
   belongs_to :need
   belongs_to :user
 
+  enum category: { 'Note': 'note',
```

Comment:
Is there any reason to store `'note'` in the database, instead of leaving it blank/Nil?

Reply:
Just seemed like a more concrete abstraction. I'll update to be nil